.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Same as above.

Can we remove the .vscode files which got uploaded
Please add the following entry in .gitignore -> `.vscode`.